index.js

```diff
@@ -1,3 +1,5 @@
+const queueTick = require('queue-tick')
```

Comment:
use var here so we don't need to major it

Reply:
Updated.